1-mandatory/2-people-i-know.js

```diff
@@ -378,13 +378,23 @@ First, I want you to find all of my friends who are 35 or older.
 */
 
 let thirtyFiveOrOlder = [];
+for (let i = 0; i < friends.length; i++) {
+  if (friends[i].age >= 35) {
+    thirtyFiveOrOlder.push(friends[i]);
+  }
+}
```

Comment:
You have done this with loop but I have used .filter() method.
let thirtyFiveOrOlder = friends.filter(friend => friend.age >= 35);

Reply:
Thank you Saliha for your feedback!